ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -1,4 +1,12 @@
 base_images:
+  old-bundle:
```

Comment:
Lets be specific here and name this `4.13-bundle` or `previous-bundle`. A follow up question is what happens when we move to 4.15? Do we have to manually fix this up?